pii-reporting-ui/src/app/features/pii-page-card/pii-card-expanded.component.css

```
@@ -0,0 +1,468 @@
+  display: flex;
+  padding: 8px 20px;
+  align-items: center;
+  border-top: 1px solid var(--sentinel-border-badge);
+}
+
+/* PII type MultiSelect - compact, black theme */
+.pii-type-multiselect {
+  max-width: 14rem;
+}
+
+.pii-type-multiselect ::ng-deep .p-multiselect {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: ng-deep

(lint/correctness/noUnknownPseudoElement)